src/commands/IgnsCommand.ts

```diff
@@ -0,0 +1,91 @@
+import { SlashCommandBuilder, ChatInputCommandInteraction } from "discord.js";
+import { Command } from "./CommandInterface";
+import { prismaClient } from "../database/prismaClient";
+import { createIGNListEmbed } from "../util/EmbedUtil";
+import { MojangAPI } from "../api/MojangAPI";
+
+export default class IgnsCommand implements Command {
+  data: SlashCommandBuilder;
+  name: string;
+  description: string;
+
+  constructor() {
+    this.name = "ign";
+    this.description = "View or modify your in-game names.";
+
+    this.data = new SlashCommandBuilder()
+      .setName(this.name)
+      .setDescription(this.description)
+      .addSubcommand((subcommand) => {
+        return subcommand
+          .setName("add")
+          .setDescription("Add an ign")
+          .addStringOption((option) => {
+            return option
+              .setName("ign")
+              .setDescription("The ign to add")
+              .setRequired(true);
```

Comment:
why would a player need to add an ign? we should just record whatever igns their discord id has used in the past in sign ups as the record